marimo/_runtime/runtime.py

```diff
@@ -2190,18 +2190,30 @@ async def preview_sql_table(self, request: PreviewSQLTableRequest) -> None:
             ).broadcast()
             return
 
-        if isinstance(engine, SQLAlchemyEngine):
-            table = engine.get_table_details(table_name, schema_name)
-            if table is None:
-                LOGGER.warning("Table %s not found", table_name)
+        try:
+            # Schemaless engines
+            if isinstance(engine, ClickhouseServer):
+                table = engine.get_table_details_from_db(
+                    database=database_name, table_name=table_name
+                )
+            else:
+                table = engine.get_table_details(
```

Comment:
i think this should just be:
```
get_table_details(database_name, schema_name, table_name)
```

then we:
1) don't need to check what kind of engine
2) the engine and implement it however (whether it will use the schema or now)